### PR TITLE
Add discord information

### PR DESCRIPTION
**Pull request by camlafit**
Explain how to user discord puprle gateway

### CODE REVIEW

File: documentation/backends/libpurple.md [hunk 1]
```diff
@@ -66,3 +66,23 @@ Some purple options are available to this specific backend :
 - Mattermost account are set in mattermost group
 - Use purple.use-mmauthtoken = 1 , if mattermost server use gitlab authenication
 - Some trouble with channels [https://github.com/EionRobb/purple-mattermost/issues/60](https://github.com/EionRobb/purple-mattermost/issues/60) and [https://github.com/SpectrumIM/spectrum2/issues/366](https://github.com/SpectrumIM/spectrum2/issues/366)
+
+
+### Notes on Mattermost support
+
+To use disccord pidging from debian, backports packages is required
```
Comment (vitalyster) [inline documentation/backends/libpurple.md]: typo?

File: documentation/backends/libpurple.md [hunk 2]
```diff
@@ -66,3 +66,23 @@ Some purple options are available to this specific backend :
...
+	
+Purple options are 
+       show-custom-emojis = [1|0]
+       display-images = [0|1]
+       open-chat-on-mention = [1|0]
+       display-images-large-servers = [0|1]
+       populate-blist = [1|0]
+       use-status-as-game = [0|1]
+       use-status-as-custom-status= [1|0]
+       disable-compress = [0|1]
```
Comment (vitalyster) [inline documentation/backends/libpurple.md]: Please format commands and options as preformatted text/code

File: documentation/backends/libpurple.md [hunk 3]
```diff
@@ -66,3 +66,23 @@ Some purple options are available to this specific backend :
 - Mattermost account are set in mattermost group
 - Use purple.use-mmauthtoken = 1 , if mattermost server use gitlab authenication
 - Some trouble with channels [https://github.com/EionRobb/purple-mattermost/issues/60](https://github.com/EionRobb/purple-mattermost/issues/60) and [https://github.com/SpectrumIM/spectrum2/issues/366](https://github.com/SpectrumIM/spectrum2/issues/366)
+
+
+### Notes on Mattermost support
```
Comment (vitalyster) [inline documentation/backends/libpurple.md]: We are discussing Discord, not Mattermost, right?

File: documentation/backends/libpurple.md (hunk 1 above)
Comment (camlafit) [reply documentation/backends/libpurple.md]: solve on next commit

File: documentation/backends/libpurple.md (hunk 2 above)
Comment (camlafit) [reply documentation/backends/libpurple.md]: should be solved on next commit

File: documentation/backends/libpurple.md (hunk 3 above)
Comment (camlafit) [reply documentation/backends/libpurple.md]: Yes was a bad copy/paste :)
Solved normaly

File: documentation/backends/libpurple.md [hunk 4]
```diff
@@ -66,3 +67,27 @@ Some purple options are available to this specific backend :
...
+
+### Notes on Discord support
+
+To use discord from debian buster, backports packages is required 
+
+	apt install purple-discord -t buster-backports
+	
+To manage connexion and accept captcha link, [harmony client](https://github.com/taylordotfish/harmony) must be installed on the server. Debian doesn't provide an updated version. Script must be installed from source.
+
+        apt install python3-pip libcairo2-dev
```
Comment (vitalyster) [inline documentation/backends/libpurple.md]: Wrong indentation?

**Comment by camlafit** [comment]
Hello

Actually I'll not update this PR. I've no more information to add :/

**Comment by vitalyster** [comment]
Still seeing indentation issues
issues

File: documentation/backends/libpurple.md [hunk 5]
```diff
@@ -66,3 +67,27 @@ Some purple options are available to this specific backend :
...
 - Some trouble with channels [https://github.com/EionRobb/purple-mattermost/issues/60](https://github.com/EionRobb/purple-mattermost/issues/60) and [https://github.com/SpectrumIM/spectrum2/issues/366](https://github.com/SpectrumIM/spectrum2/issues/366)
+
+
+### Notes on Discord support
+
+To use discord from debian buster, backports packages is required 
+
+	apt install purple-discord -t buster-backports
+	
+To manage connexion and accept captcha link, [harmony client](https://github.com/taylordotfish/harmony) must be installed on the server. Debian doesn't provide an updated version. Script must be installed from source.
```
Comment (vitalyster) [inline documentation/backends/libpurple.md]: > connexion

Typo

**Review by vitalyster — CHANGES_REQUESTED** [review]

**Comment by camlafit** [comment]
Hello

Should be solve with last commit.

Thanks

**Review by vitalyster — APPROVED** [review]

**Pull request merged**